src/styles/container.scss

```diff
@@ -0,0 +1,77 @@
+.watch-container {
+  @include square($watch-container-size);
+  position: relative;
+  background-color: rgb(239, 237, 252);
+  margin: auto;
```

Comment:
you can use wrapper 😉

Reply:
it is forbidden to set outer styles for BEM block, so you just need some outer block and add its element class on your current block to be able to set `margin` 😉